### DISCUSSION TITLE
Understrap 1.2-beta1 Coming Soon

### DISCUSSION THREAD

**Discussion by bacoords** [holger1411/understrap#1917] [Announcements]
Our plan is to tie up any PRs that are close to completion and tag a Beta version of Understrap (1.2-beta1) in the next week (so late September). 

Mainly with the [WooCommerce templates](https://github.com/understrap/understrap/pull/1904) being out of date, now is a good time to wrap up everything and prepare to test and then launch it. 

Any newer, completely unreviewed, or draft/wip PRs will not be merged until 1.3 as we work on releasing 1.2. If there's anything critical we think needs to go out in 1.2 please comment here as this is all open to discussion.

CC @IanDelMar @howarddc

**Comment by IanDelMar** [comment]
You could add Milestones to the PRs. Then people know which PR is planned to be part of which future release.

Ping me if conflicts arise in the process of merging PRs.

**Reply by bacoords** [reply]
Sounds good! - I'm milestoning everything that seems reasonable for 1.2, then anything else that seems logical to merge but maybe not immediately for 1.3

**Comment by johnmontfx** [comment]
Hey Brian -- is it appropriate to test 1.2 using the existing child theme? I'm currently in active development on a site and it would provide a good opportunity to test.

**Reply by bacoords** [reply]
Yes definitely! If you're using a child theme that will eventually allow the parent theme to update, then you can definitely test the beta now.

**Reply by johnmontfx** [reply]
OK...I'll switch over dev to that today. thanks!
